Real_world_examples/Water_quality_suspended_matter.ipynb

```diff
@@ -0,0 +1,960 @@
+{
```

Comment:
<p>Just noting that the "Last Tested" section is not in other DE Australia Notebooks? Do we want to do add it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
I have removed the 'last tested' section